playgrounds/petclinic-maven-sb-2/src/main/resources/templates/error.html

```
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+
+<html xmlns:th="https://www.thymeleaf.org" th:replace="~{fragments/layout :: layout (~{::body},'error')}">
+
+<body>
+  <img src="../static/resources/images/pets.png" th:src="@{/resources/images/pets.png}" />
```

HTMLHint (1.9.2)

[warning] An alt attribute must be present on <img> elements.

(alt-require)